interop/xds/server/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 gRPC authors.
```

Comment:
Take a look at my review for the client side: https://github.com/grpc/grpc-go/pull/4443#pullrequestreview-660290241. Most comments apply here as well.

Reply:
Done. Thanks.